comp/otelcol/otlp/components/exporter/serializerexporter/logger.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+package serializerexporter
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+)
+
+// zaplogger implements the tracelog.Logger interface on top of a zap.Logger
```

Comment:
Can you create a tix to de-dup this file and http://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1c3743853f0ec2de78bc7873ed04d900ebe8b1a7/exporter/datadogexporter/zaplogger.go ?

This should be moved to pkg/datadog and shared